eth/beacon/helpers.py

```diff
@@ -63,47 +62,49 @@
     from eth.beacon.types.validator_records import ValidatorRecord  # noqa: F401
 
 
-def _get_element_from_recent_list(
-        target_list: Sequence[Any],
-        target_slot: SlotNumber,
-        slot_relative_position: SlotNumber) -> Any:
+#
+# Get block root
+#
+def _get_block_root(
+        latest_block_roots: Sequence[Hash32],
+        state_slot: SlotNumber,
+        slot: SlotNumber,
+        latest_block_roots_length: int) -> Hash32:
     """
-    Return the element from ``target_list`` by the ``target_slot`` number,
-    where the the element should be at ``target_slot - slot_relative_position``th
-    element of the given ``target_list``.
+    Return the block root at a recent ``slot``.
     """
-    target_list_length = len(target_list)
-
-    if target_slot < slot_relative_position:
+    if state_slot > slot + latest_block_roots_length:
         raise ValueError(
-            "target_slot (%s) should be greater than or equal to slot_relative_position (%s)" %
-            (target_slot, slot_relative_position)
+            "state.slot (%s) should be less than or equal to "
+            "(slot + latest_block_roots_length) (%s), "
+            "where slot=%s, latest_block_roots_length=%s" %
```

Comment:
Can you update this to use `"some {0}".format("value")` style string formatting to be consistent with the rest of the codebase.